### PR TITLE
Use trained teacher2

### PR DESCRIPTION
**Pull request by mithunpaul08**

### CODE REVIEW

**Comment by mithunpaul08** [comment]
Merging with the branch UseTrainedTeacher2. This was an experiment tried out in April2020 to check if using a pre-trained teacher is better in a student teacher model than trying to train teacher and student together. The experiment was a failure. still just merging the code in to save the code. If you set the variable use_trained_teacher_inside_student_teacher_arch to True, it will load the logits of a previously trained model. Assumption is that you have trained the model offline and stored the prediction logits on a training data- preferably in a nfcv environment. There is also another branch here which does nfcv/which was used to create the trained model predictions on fever training partition
